src/Sulu/Bundle/AdminBundle/Admin/Routing/DatagridRouteBuilder.php

```diff
@@ -92,6 +110,20 @@ public function disableSearching(): DatagridRouteBuilderInterface
         return $this;
     }
 
+    public function enableDeletion(): DatagridRouteBuilderInterface
```

Comment:
This is not consistent with the way this is done for the forms. The forms have to add a `sulu_admin.delete` ToolbarAction to get the delete button. We can discuss which way we go, but I think it should be the same for form and datagrids.

Reply:
did change it and use addToolbarActions